api/nodes/views.py

```diff
@@ -1925,9 +1925,8 @@ def get_default_queryset(self):
             # Resolve to a provider-specific subclass, so that
             # trashed file nodes are filtered out automatically
             ConcreteFileNode = BaseFileNode.resolve_class(provider, BaseFileNode.ANY)
-            return ConcreteFileNode.objects.filter(
-                id__in=[self.get_file_item(file).id for file in files_list],
-            )
+            file_ids = [f.id for f in self.bulk_get_file_nodes_from_wb_resp(files_list)]
```

Comment:
Just verifying that there's not a better way to do this. Since this seems to be the only thing currently using the function, it could potentially output the ids. One way is, since you're going through all the items in the bulk_get_etc function, you could just construct and return the ids as a list rather than the objects.  If we're expecting a use for the actual file objects at some point, then this would be an excessive optimization.